holoviews/plotting/bokeh/annotation.py

```diff
@@ -35,9 +35,13 @@ def get_data(self, element, ranges, style):
             data = dict(x=[element.x], y=[element.y])
         self._categorize_data(data, ('x', 'y'), element.dimensions())
         data['text'] = [element.text]
-        style['text_align'] = element.halign
+        if 'text_align' not in style:
+            style['text_align'] = element.halign
         baseline = 'middle' if element.valign == 'center' else element.valign
-        style['text_baseline'] = baseline
+        if 'text_baseline' not in style:
+            style['text_baseline'] = baseline
+        if 'text_font_size' not in style:
+            style['text_font_size'] = '%dPt' % element.fontsize
```

Comment:
One day we can try to allow the user to specify font sizes in something other than points. For now this is fine though...

Reply:
Right, this is the semantics that matplotlib uses so I stuck with Pts. If you set the style option you can use a different unit.

Reply:
To be honest using points as a default is probably fine and not worth changing (or even exposing). If anything the problem is documenting that the default is in points.